components/asic/asic.c

```diff
@@ -9,13 +9,25 @@
 
 #include "asic.h"
 #include "device_config.h"
+#include "driver/gpio.h"
+
+#define GPIO_ASIC_RESET CONFIG_GPIO_ASIC_RESET
 
 static const double NONCE_SPACE = 4294967296.0; //  2^32
 
 static const char *TAG = "asic";
 
 uint8_t ASIC_init(GlobalState * GLOBAL_STATE)
 {
+    ESP_LOGI(TAG, "Initializing %s", GLOBAL_STATE->DEVICE_CONFIG.family.asic.name);
+
+    esp_rom_gpio_pad_select_gpio(GPIO_ASIC_RESET);
```

Comment:
It's debatable if this code belongs here, or outside of this component. Strictly seen it should be outside, but then the code needs to be duplicated in `main` and `self_test`. Any suggestions are welcome.